### PR TITLE
Add cron task to purge revisions once a day

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

**Comment by nlisgo** [comment]
This should not be merged in until after the revision tables have been cleared again. We have `329469` entries to purge. I will mark this "Ready for Review" once they are cleared. We should get this in as soon as possible after that.

**Comment by nlisgo** [comment]
Have a PR in play on Journal CMS to add logging to the 2 commands used here: https://github.com/elifesciences/journal-cms/pull/760

That should be merged in before this.

**Comment by nlisgo** [comment]
By my estimate this should be taking 1-2 hours a day. If we don't clear it regularly then it results in the database getting bloated.

Hopefully, this can be taken on in a better way when we move to Drupal 9.

The following has been merged: https://github.com/elifesciences/journal-cms/pull/760

So this is ready to go.

File: salt/journal-cms/cron.sls [hunk 1]
```diff
@@ -14,3 +14,13 @@ journal-cms-cron-drupal:
         - minute: "*/5"
         - require:
             - migrate-content
+
+journal-cms-cron-purge-revisions:
+    cron.present:
+        - identifier: journal-cms-cron-purge-revisions:
```
Comment (lsh-0) [inline salt/journal-cms/cron.sls:20]: drop the trailing colon

**Review by lsh-0 — APPROVED** [review]
Looks good, feel free to merge

**Pull request merged**